app/utils/transition-target-url.ts

```diff
@@ -1,4 +1,4 @@
-import Transition from '@ember/routing/transition';
+import Transition from '@ember/routing/-private/transition';
```

Comment:
Oh, good. :/

Reply:
This is a weird one. `@ember/routing/transition` was never a real module, just one invented for types.